go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/openshift/imagebuilder
 
-go 1.22.0
+go 1.23.3
```

Comment:
@nalind or @flouthoc any concerns with this Go bump?

Reply:
The openshift/builder repository is using "toolchain 1.23.1", and containers projects' main branches are on 1.23, so I think we're okay.